clients/ops/admin-ui/src/features/subject-request/manual-processing/ManualProcessingDetail.tsx

```diff
@@ -0,0 +1,184 @@
+    if (!hasChanged) return;
+    const params: PatchUploadManualWebhookDataRequest = {
+      connection_key: data.connection_key,
+      privacy_request_id: data.privacy_request_id,
+      body: { ...values } as object,
+    };
+    onSaveClick(params);
+    onClose();
+  };
+
+  return (
+    <>
+      {data?.checked && (
+        <Button
+          color="gray.700"
+          fontSize="xs"
+          h="24px"
+          onClick={onOpen}
+          variant="outline"
+          w="58px"
+        >
+          Review
+        </Button>
+      )}
+      {!data?.checked && (
+        <Button
+          color="white"
+          bg="primary.800"
+          fontSize="xs"
+          h="24px"
+          onClick={onOpen}
+          w="127px"
+          _hover={{ bg: "primary.400" }}
+          _active={{ bg: "primary.500" }}
+        >
+          Begin manual input
+        </Button>
+      )}
+      <Drawer
+        isOpen={isOpen}
+        placement="right"
+        // @ts-ignore
+        initialFocusRef={firstField}
+        onClose={onClose}
+        size="lg"
+      >
+        <DrawerOverlay />
+        <DrawerContent>
+          <DrawerCloseButton />
+          <DrawerHeader color="gray.900">
+            <Text fontSize="xl" mb={4}>
+              {connectorName}
+            </Text>
+            <Divider />
+            <Text fontSize="md" mt="8">
+              PII Requirements
+            </Text>
+            <Box mt="8px">
+              <Text color="gray.700" fontSize="sm" fontWeight="normal">
+                Please complete the following PII fields that have been
+                collected for the selected subject.
+              </Text>
+            </Box>
+          </DrawerHeader>
+          <DrawerBody mt="24px">
+            <Formik
+              initialValues={{ ...data.fields }}
+              onSubmit={handleSubmit}
+              validateOnBlur={false}
+              validateOnChange={false}
+              validationSchema={Yup.object().shape({})}
+            >
+              <Form id="manual-detail-form" noValidate>
+                <VStack align="stretch" gap="16px">
+                  {Object.entries(data.fields).map(([key], index) => (
+                    <HStack key={key}>
+                      <Field id={key} name={key}>
+                        {({ field }: { field: any }) => (
+                          <FormControl
+                            alignItems="baseline"
+                            display="inline-flex"
+                          >
+                            <FormLabel
+                              color="gray.900"
+                              fontSize="14px"
+                              fontWeight="semibold"
+                              htmlFor={key}
+                              w="50%"
+                            >
+                              {key}
+                            </FormLabel>
+                            <Input
+                              {...field}
```

Comment:
Input field works well, tested things like `"` and `✊🐝  Ꮆя𝐞ＥｔⓘŇｇ𝓢 ℱ𝔯Øϻ σᑌ𝓉𝕖𝕣 ѕρᗩ匚ᵉ  🍭` empty fields and random code. 

Wanted to get your opinion, should we limit the text we can upload here? Since you need to be able to login to fidesops I doubt people would paste novels into this block.

Reply:
@pattisdr Great suggestion. What are the current DB field constraints for these? Once I have the character limitiations, I can implement the client side validation accordingly.

Reply:
I didn't add DB constraints on these particular fields, I think if we did have limits they should be fairly high.